src/backends/perf_pt/collect.c

```diff
@@ -227,6 +227,10 @@ poll_loop(int perf_fd, int stop_fd, struct perf_event_mmap_page *mmap_hdr,
             goto done;
         }
 
+        // POLLIN on pfds[0]: Overflow event on either the Perf AUX or data buffer.
+        // POLLHUP on pfds[1]: Tracer stopped by parent.
+        // It's important we read any remaining trace data out of the AUX
```

Comment:
Why is it important?

Reply:
Actually, since I raised this PR I've learned more, so we can probably kill this part of the comment.